package.json

```diff
@@ -67,5 +76,13 @@
     "url-loader": "^2.1.0",
     "webpack": "^4.39.3",
     "webpack-dev-server": "^3.8.0"
+  },
+  "engines": {
+    "node": "^10.16.3"
```

Comment:
This turned out to break my env. I upgraded to node 10.16 and now I'm getting 

`avn could not activate node v10.16.3`

and I removed and reinstalled `avg` et al but still get this error :(